.github/workflows/integration-tests.yml

```diff
@@ -87,16 +85,14 @@ jobs:
       with:
         python-version: 3.7
     - name: Setup chartpress
-      env:
-        DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
-        DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
       run: |
         pip install --user pipx
         pipx ensurepath
         pipx install pipenv
         pipenv install --dev
-        echo ${DOCKER_PASSWORD} | docker login -u ${DOCKER_USERNAME} --password-stdin
-        pipenv run chartpress --push
+        pipenv run chartpress
+        kind load docker-image $(pipenv run chartpress --list-images) || true
```

Comment:
I think it would be good to open an issue on the kind repo (and maybe add a comment here that links to the issue).

Reply:
@ableuler I submitted an issue with kind here: https://github.com/kubernetes-sigs/kind/issues/2477

After trying to reproduce on my laptop I see that trying to load the images in kind all at once causes an error. Whereas trying to load the images withe separate commands, one by one, goes just fine.